ckanext/unaids/theme/templates/user/edit_user_form.html

```diff
@@ -6,7 +6,7 @@
   <fieldset>
     <legend>{{ _('Change details') }}</legend>
 
-    <p>If you need to change any disabled field, please contact <a href="mailto:estimates@unaids.org">estimates@unaids.org</a> for assistance.</p>
+      <p>{{ _('To change your profile settings, please') }} <a href="{{ h.get_ape_url() }}">{{ _('click here') }}</a>.</p>
```

Comment:
looks like missing space after ", please".

Reply:
Also this change should include changes i po/mo files.
The link to APE app should include the locales too so that the form is displayed in correct lang (fr, pt etc.)